test/prefer-spread.js

```diff
@@ -146,7 +146,11 @@ test.snapshot({
 		outdent`
 			const string = 'foo';
 			foo = string.concat("bar");
-		`
+		`,
+		// #1068
+		'const bufA = Buffer.concat([buf1, buf2, buf3], totalLength);',
+		'Foo.concat(1)',
+		'A.concat(1)'
```

Comment:
Can you also add `FooBar.concat(1)`?

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/pull/1069/commits/0a1e32361fb25c4b2ce6f5529d5c3a4f5b8bb01b